src/emulator/download.js

```diff
@@ -21,6 +26,7 @@ module.exports = (name) => {
     });
     req.on("end", () => {
       writeStream.close();
```

Comment:
consider using a checksum to verify before copy/run

Reply:
great suggestion, will do